src/main/java/com/blamejared/jeitweaker/zen/JeiManager.java

```diff
@@ -0,0 +1,115 @@
+package com.blamejared.jeitweaker.zen;
+
+import com.blamejared.crafttweaker.api.CraftTweakerAPI;
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker.api.fluid.IFluidStack;
+import com.blamejared.crafttweaker.api.item.IItemStack;
+import com.blamejared.crafttweaker.api.managers.IRecipeManager;
+import com.blamejared.crafttweaker.impl.util.text.MCTextComponent;
+import com.blamejared.crafttweaker_annotations.annotations.Document;
+import com.blamejared.jeitweaker.actions.ActionAddInfo;
+import com.blamejared.jeitweaker.actions.ActionAddIngredient;
+import com.blamejared.jeitweaker.actions.ActionAddRecipeCategory;
+import com.blamejared.jeitweaker.actions.ActionHideCategory;
+import com.blamejared.jeitweaker.actions.ActionHideIngredient;
+import com.blamejared.jeitweaker.actions.ActionHideMod;
+import com.blamejared.jeitweaker.actions.ActionHideRecipe;
+import com.blamejared.jeitweaker.actions.ActionHideRegex;
+import com.blamejared.jeitweaker.zen.category.JeiCategory;
+import com.blamejared.jeitweaker.zen.component.HackyJeiIngredientToMakeZenCodeHappy;
+import com.blamejared.jeitweaker.zen.component.IFluidStackExpansions;
+import com.blamejared.jeitweaker.zen.component.IItemStackExpansions;
+import net.minecraft.util.ResourceLocation;
+import org.openzen.zencode.java.ZenCodeType;
+
+import java.util.Arrays;
+
+@Document("mods/JEITweaker/JEI")
```

Comment:
This can be `mods/JEI/JEI`. I have been meaning to rename the folder for a while